src/components/Contacts/ContactDetails/ContactDetailsTab/People/ContactDetailsTabPeople.tsx

```diff
@@ -46,6 +56,27 @@ const ContactPersonIconContainer = styled(Box)(() => ({
   marginRight: '35px',
 }));
 
+const ContactDetailEditIcon = styled(CreateIcon)(({ theme }) => ({
+  width: '18px',
+  height: '18px',
+  margin: theme.spacing(0),
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+const ContactAddIcon = styled(AddIcon)(() => ({
+  color: '#2196F3',
```

Comment:
Are any of these theme colors?

Reply:
It's not, but maybe it should be since it seems to be getting used in more and more places?